tests/cpp_unit_tests/test_line.cpp

```diff
@@ -202,8 +197,9 @@ TEST_CASE("Test line") {
         CHECK(output.q_to(1) == doctest::Approx(imag(s_t) / 3.0));
     }
 
-    SUBCASE("Asymmetric short circuit results") {
-        BranchShortCircuitOutput output = branch.get_sc_output<false>(if_sc_asym, it_sc_asym);
+    SUBCASE("Short circuit results") {
+        BranchShortCircuitOutput sym_output = branch.get_sc_output(if_sc, it_sc);
+        BranchShortCircuitOutput output = branch.get_sc_output(if_sc_asym, it_sc_asym);
```

Comment:
`asym_output`? if not, please create a new `SUBCASE`

(also for the other files)

Reply:
Made it in different subcases